deployment/ansible/roles/district-builder.app-server/tasks/main.yml

```diff
@@ -0,0 +1,51 @@
+---
+# tasks file for districtbuilder.app-server
+- name: Disable ECS Agent
+  service: name=ecs state=stopped
+
+- name: Configure DistrictBuilder service
+  template: src="{{ item.src }}" dest="{{ item.dest }}"
+  with_items:
+    - { src: "logrotate.conf.j2", dest: "/etc/logrotate.d/district-builder" }
+    - { src: "district-builder.service.j2", dest: "/etc/init/district-builder.conf" }
+  notify:
+    - Restart DistrictBuilder
+
+- name: Install docker-compose
+  pip: name=docker-compose version="{{ docker_compose_version }}" state=present
+
+- name: Create App Files
+  file: path="{{ item.path }}" state="{{ item.state }}" mode="{{ item.mode }}" owner="{{ ansible_user }}" recurse="{{ item.recurse | default(omit) }}"
+  with_items:
+    - { 
+        path: "{{ districtbuilder_app_home }}/scripts",
+        state: "directory",
+        mode: "0755",
+        recurse: true
+      }
+    - {
+        path: "{{ districtbuilder_app_home }}/user-data/config_settings.py",
+        state: "touch",
+        mode: "0644"
+      }
+
+- name: Create docker-compose configuration
+  template: src="{{ item.src }}" dest="{{ item.dest }}"
+  with_items:
+    - { src: "docker-compose.yml.j2", dest: "{{ districtbuilder_app_home }}/docker-compose.yml" }
+    - { src: "env.conf.j2", dest: "{{ districtbuilder_app_home }}/.env" }
+  notify:
+    - Restart DistrictBuilder
+
+- name: Update settings, run migrations, update geoserver password
+  command: "{{ item }}"
+  args:
+    chdir: "{{ districtbuilder_app_home }}"
+  with_items:
+    - docker-compose pull
```

Comment:
It looks like `district-builder.app-server` does what `scripts/update` does now except that it pulls the latest images instead of building them. If we change this in one place will we have to remember to change it in both places?

Reply:
Yeah, we will. I'll make a note of that in `scripts/update`.

> It looks like district-builder.app-server does what scripts/update does now except that it pulls the latest images instead of building them.

The other difference is that `scripts/update` also expects a `postgres` container to exist, and it  _has_ to be brought up explicitly before `django` so that the health check can run. That breaks things in Production (we use AWS RDS for the DB there), so I had to create a separate `update` script.